### ISSUE TITLE
CVODE detection fails on Appveyor, but simulation does work

### ISSUE THREAD

**Issue by MichaelClerx** [MichaelClerx/myokit#237]
Output from Appveyor for Python2.7:

https://ci.appveyor.com/project/MichaelClerx/myokit/builds/20057698/job/kxknixceof7qmfrk

```
%PYTHON%\python.exe -m myokit system
WARNING:myokit:Using development version of Myokit. This may contain untested features and bugs. Please see http://myokit.org for the latest stable release.
WARNING:myokit._config:Unable to auto-detect Sundials version.
WARNING:myokit._config:Sundials version not set in myokit.ini and version auto-detection failed.
== System information ==
Myokit: 1.27.8.dev
Python: 2.7.15 (v2.7.15:ca079a3ea3, Apr 30 2018, 16:30:26) [MSC v.1500 64 bit (AMD64)]
OS: Windows (win32, nt)
== Python requirements ==
NumPy: 1.15.4
SciPy: 1.1.0
Matplotlib: 2.2.3
ConfigParser: OK
Setuptools: 40.5.0
== Python extras ==
SymPy: OK
CMAES: Not found
MoviePy: Not found
== GUI ==
PyQt5: Not found
PyQt4: Not found
PySide: Not found
PySide2: Not found
== Development tools ==
Sphinx: Not found
Flake8: Not found
== Simulation tools ==
Compiler: Microsoft Visual Studio 9.0 (2008) - 150030729
Sundials: Not found
OpenCL: No OpenCL support detected.
```

**Comment by MichaelClerx** [comment]
Doesn't work for 3.6 either (with a much newer compiler)

https://ci.appveyor.com/project/MichaelClerx/myokit/builds/20057698/job/pxl54pfmw5ak86ka

Still guessing this is a compilation error

**Comment by MichaelClerx** [comment]
Can recreate this on my own windows machine

**Comment by MichaelClerx** [comment]
Embedded sundials binaries/headers for windows weren't loading when creating ini file, only when loading afterwards. Fixing that seems to have fixed this:

```
== System information ==
Myokit: 1.27.8.dev
Python: 2.7.15 (v2.7.15:ca079a3ea3, Apr 30 2018, 16:30:26) [MSC v.1500 64 bit (AMD64)]
OS: Windows (win32, nt)
== Python requirements ==
NumPy: 1.15.4
SciPy: 1.1.0
Matplotlib: 2.2.3
ConfigParser: OK
Setuptools: 40.5.0
== Python extras ==
SymPy: OK
CMAES: Not found
MoviePy: Not found
== GUI ==
PyQt5: Not found
PyQt4: Not found
PySide: Not found
PySide2: Not found
== Development tools ==
Sphinx: Not found
Flake8: Not found
== Simulation tools ==
Compiler: Microsoft Visual Studio 9.0 (2008) - 150030729
Sundials: 2.7.0
OpenCL: No OpenCL support detected.
```

**Comment by MichaelClerx** [comment]
Closed in #238
